appveyor.yml

```diff
@@ -0,0 +1,27 @@
+version: 1.0.0.{build}
+image:
+  - Visual Studio 2017
+pull_requests:
+  do_not_increment_build_number: true
+branches:
+  only:
+    - master
+skip_branch_with_pr: true
+configuration: Release
+#before_build:
+#  - cmd: dotnet restore
+build:
+  verbosity: minimal
+  parallel: true
+build_script:
+  - cmd: dotnet build -c Release
+test_script:
+  - cmd: dotnet test -c Release
+artifacts:
+  - path: '**\*.nupkg'
+nuget:
+  disable_publish_on_pr: true
+#deploy:
```

Comment:
@JohannesMoersch Better to use this automatic nuget publish config over travis one : )